src/main/java/com/internet/shop/model/User.java

```diff
@@ -22,6 +23,21 @@ public User(Long id, String name, String login, String password) {
         this.password = password;
     }
 
+    public User(String name, String login, String password, byte[] salt) {
+        this.name = name;
+        this.login = login;
+        this.password = password;
+        this.salt = salt;
+    }
+
+    public User(Long id, String name, String login, String password, byte[] salt) {
```

Comment:
there are too many constructors at the moment, do we need them all?

Reply:
removed